omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -804,4 +809,27 @@ function PatientSearchWidget(configuration){
         // For some reason without this the cursor is at position 0 of the input, i.e. before the initial value
         input[0].selectionStart = input.val().length;
     }
+
+
+    // Store the original XMLHttpRequest open function to preserve its functionality
+    const originalXHR = window.XMLHttpRequest.prototype.open;
+
+    // Override the XMLHttpRequest open function to intercept specific requests
+    window.XMLHttpRequest.prototype.open = function (method, url) {
+
+        // Check if the requested URL contains '/openmrs/registrationapp/biometrics/biometrics/search.action'
+        if (url.includes('/openmrs/registrationapp/biometrics/biometrics/search.action')) {
```

Comment:
This should not check for the leading "/openmrs", as the application may be deployed with a different webapp name than openmrs.

Reply:
instead of displaying it every time a “searchOnIdentifier” is triggered, I display it when the search:clear event is triggered